### ISSUE TITLE
Remove queryable kind

### ISSUE THREAD

**Issue by OlivierHecart** [eclipse-zenoh/zenoh#236]
The queryable kind:
- Makes the wire protocol and the routing more complex.
- Brings extra concepts (like eval) and complexity to the API which are not necessary as:
- If one really wants to discriminate stored values and computed values he should use different key expressions.

So we should remove the queryable kind.

**Comment by Mallets** [comment]
6971f978e09524f747fc69d5d5cdf7eb825ac536 and 59d6b90306c377982e45c2941b935a5efddf436a have been merged into api-changes branch.
This issue will need to be closed manually once api-changes is merged into master.

**Comment by Mallets** [comment]
Merged into master.
